lib/workers/repository/init/index.ts

```diff
@@ -12,13 +12,42 @@ import { mergeRenovateConfig } from './config';
 import { detectSemanticCommits } from './semantic';
 import { detectVulnerabilityAlerts } from './vulnerability';
 
+let cache: repositoryCache.Cache;
+
 function initializeConfig(config: RenovateConfig): RenovateConfig {
   return { ...clone(config), errors: [], warnings: [], branchList: [] };
 }
 
 async function initializeCaches(config: RenovateConfig): Promise<void> {
   memCache.init();
   await repositoryCache.initialize(config);
+  cache = repositoryCache.getCache();
+  cache.init = cache.init || {};
+}
+
+// istanbul ignore next
```

Comment:
Are you sure to ignore this? Do we have a way to test it? I don't want to have a broken renovate again today 🙃

Reply:
True, I could at least export this to unit test it